src/shell/dotnet/Shell/Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "Shell": {
+      "commandName": "Project"
+    },
+    "Chart and grid": {
+      "commandName": "Project",
+      "commandLineArgs": "--url http://localhost:4200"
```

Comment:
Does this result in attempting to serve both web apps (chart & grid) on the same port?

Reply:
No, this is starting the Shell assuming the grid and chart examples are running.